WeakAuras/RegionTypes/RegionPrototype.lua

```diff
@@ -3,6 +3,67 @@ local L = WeakAuras.L;
 
 WeakAuras.regionPrototype = {};
 
+-- Adjusted Duration
+
+function WeakAuras.regionPrototype.AddAdjustedDurationToDefault(default)
+  default.useAdjustededMax = false;
+  default.useAdjustededMin = false;
+end
+
+function WeakAuras.regionPrototype.AddAdjustedDurationOptions(options, data, order)
```

Comment:
It should be in WeakAurasOptions, but there's no RegionPrototype file for the Options yet. And I didn't create it for this. Long term plan is to unduplicate the text options and then it would make sense to have a separate file for the common options.

The WeakAuras code is in general not that well-structured and so my concern is mainly if I'm improving the code organization.